net-mgmt/snmp/pkg-descr

```diff
@@ -0,0 +1,6 @@
+ bsnmp. It allows you to use SNMPv1 and SNMPv2 to query your OPNsense appliance
```

Comment:
yes, exactly my point, they may have useful info in them that could be used as well